.github/workflows/test_cloudpods.yml

```diff
@@ -54,19 +54,13 @@ jobs:
           python-version: '3.9'
 
       - name: Start LocalStack
+        uses: LocalStack/setup-localstack@main
+        with:
+          image-tag: ${{ matrix.tag }}
+          use-pro: 'true'
+          install-awslocal: 'true'
         env:
-          DEBUG: 1 
-          POD_LOAD_CLI_TIMEOUT: 300
```

Comment:
is there a reason we completely remove this configuration? I think debugging would still be nice in case the workflow fails (we do print the logs in that case). Not sure about the pod-load config, I can't find in the docs anymore.

Reply:
You are absolutely right. Thanks for spotting it. Regarding `POD_LOAD_CLI_TIMEOUT`, I try to find out what's the case.